recipe/meta.yaml

```diff
@@ -20,10 +20,7 @@ package:
 source:
   fn: {{ name }}-{{ version }}.tar.gz
   url: http://ftp.mcs.anl.gov/pub/fathom/{{ name }}-{{ version }}.tar.gz
-  sha256: 60d31762be3f0e5c89416c764e844ec88dac294169b59a5ead3c316b50f85c29
-  patches:
-    - 0001-tempestremap-serial.patch
-
+  sha256: 51c31ccbcaa76d9658a44452b9a39f076b795b27a1c9f408fc3d0eea97e032ef
```

Comment:
@vijaysm - can we use the tarball published on bitbucket as the official release?  In fact, why store a separate tarball given the (admittedly small!) risk of them diverging...?

Reply:
I'll hold off on merging until @vijaysm has a chance to answer this.

I believe the answer is that `autotools` gets run on the source from bitbucket so that folks don't have to install autotools themselves.  @vijaysm, is that right?

Reply:
Yes @xylar is correct in that the tarball available from the ftp site is the result of a `make distcheck` operation, which includes all the configuration information required to build with autotools, without the user having to use `autoreconf -fi` themselves. The tarball from bitbucket is essentially a snapshot of the sources in that particular hash, and nothing else. 

So it's more of a matter of convenience for release tarballs that depends on autotools. When we fully transition to pure CMake build system, we can avoid this divergence :-)

Reply:
It might make sense to switch to building with CMake here.  Probably not with this release because we didn't test it with the `rc`s.  But next release, maybe?